packages/wallets/src/features/cfd/screens/CompareAccounts/constants.ts

```diff
@@ -0,0 +1,53 @@
+export const CFD_PLATFORMS = {
```

Comment:
Pls rename this, it has the same name as shared package. And pls put it in constants.tsx. Dont scope this just for compare accounts.

Reply:
We are not using shared package in our wallets project and it's the same thing so i don't think renaming it makes sense. Moved it to parent constants.tsx file